changelog.txt

```diff
@@ -2,6 +2,7 @@
  * Added Realm.isEmpty().
  * Fixed a bug making it impossible to convert a field to become required during a migration (#1695).
  * Fixed a bug making it impossible to read Realms created using primary keys and created by iOS (#1703).
+ * Fixed a bug cached column indexes were cleared improperly.
```

Comment:
"Fixed a bug causing cached field indexes to be cleared to soon"
Also add a reference to the issue or PR